app/controllers/ViewsController.scala

```diff
@@ -34,7 +35,7 @@ class ViewsController(val acl: Acl, assetsManager: AssetsManager, isDev: Boolean
   }
 
   def collectionEditor(priority: String) = getCollectionPermissionFilterByPriority(priority, acl)(ec) { request =>
-    if (shouldRedirectToV2(request)) {
+    if (priority != "breaking-news" && shouldRedirectToV2(request)) {
```

Comment:
I'd like the priority check moved into the shouldRedirect function.

Reply:
👍